projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-component.scss

```diff
@@ -390,6 +411,26 @@
         @extend %igx-grid__filtering-scroll-end !optional;
     }
 
+    @include e(hierarchical-indent) {
+        @extend %igx-grid__hierarchical-indent !optional;
+    }
+
+    @include e(hierarchical-expander) {
+        @extend %igx-grid__hierarchical-expander !optional;
+    }
+
+    @include e(hierarchical-expander, $m: header) {
+        @extend %igx-grid__hierarchical-expander--header !optional;
+    }
+
+    @include e(hierarchical-expander, $m: push) {
+        @extend %igx-grid__hierarchical-expander--push !optional;
+    }
+
+    @include e(hierarchical-indent, $m: scroll) {
```

Comment:
If these are all related to the hierarchical grid, why are they not elements of the igx-grid-hierarchical block element?

Reply:
We don't have igx-grid-hierarchical as a block. We already talk about that. My first approach was to make it as a separate block, but then I speak with you, and we agree that it's not necessary to be a separate block. I forgot to remove the block igx-grid-hierarchical from the sass component, maybe you saw that, and that's why you think it should be part of it, but the block is useless the placeholder that it holds doesn't exist, it's a garbage code which I will remove.

Reply:
Remove it then.